gen.go

```diff
@@ -22,56 +23,105 @@ import (
 )
 
 const (
-	// connectionFile  string = "pkg/gen/connection.go"
+	connectionFile  string = "connection.go"
 	enumFile        string = "enum.go"
 	inputObjectFile string = "input.go"
-	// interfaceFile   string = "pkg/gen/interface.go"
-	// mutationFile    string = "pkg/gen/mutation.go"
-	// objectFile      string = "pkg/gen/object.go"
-	// payloadFile     string = "pkg/gen/payload.go"
-	// queryFile       string = "pkg/gen/query.go"
-	// scalarFile      string = "pkg/gen/scalar.go"
-	// unionFile       string = "pkg/gen/union.go"
+	interfacesFile  string = "interfaces.go"
+	objectFile      string = "object.go"
+	queryFile       string = "query.go"
+	mutationFile    string = "mutation.go"
+	payloadFile     string = "payload.go"
+	// scalarFile      string = "scalar.go" // NOTE: probably not useful
```

Comment:
Would prefer if we just deleted these now

Reply:
There is a good chunk of work left in `gen.go`. I'm not 100% sure that these parts are useless so I'll wait to remove for now

Reply:
got it